docs/azure/sdk/authentication/additional-methods.md

```diff
@@ -1,20 +1,20 @@
 ---
-title: Additional methods to authenticate to Azure resources from .NET apps
+title: Additional methods to authenticate to Azure from .NET apps
 description: This article describes additional, less common methods you can use to authenticate your .NET app to Azure resources. 
 ms.topic: how-to
 ms.custom: devx-track-dotnet, engagement-fy23
-ms.date: 2/28/2023
+ms.date: 07/31/2024
 ---
 
 # Additional methods to authenticate to Azure resources from .NET apps
 
-This article lists additional methods apps may use to authenticate to Azure resources. The methods on this page are less commonly used and when possible, it is encouraged to use one of the methods outlined in the [authenticating .NET apps to Azure using the Azure SDK overview](./index.md) article.
+This article lists additional methods apps may use to authenticate to Azure resources. The methods on this page are less commonly used; when possible, we encourage you to use one of the methods outlined in [authenticating .NET apps to Azure using the Azure SDK overview](./index.md).
```

Comment:
```suggestion
This article lists additional methods apps may use to authenticate to Azure resources. The methods on this page are less commonly used; when possible, use one of the methods outlined in [authenticating .NET apps to Azure using the Azure SDK overview](./index.md).
```

Optional nit: I don't think the "we" perspective/voice is usually used in Azure docs